RELEASE.txt

```diff
@@ -26,7 +26,11 @@ This document describes the differences between v6.1.0 and v6.0.0.
 - refactoring (rxr)
 - fixed patsetbit and patsetmask for moench
 - changed default vref of adc9257 to 2V for moench (from 1.33V)
+<<<<<<< HEAD
```

Comment:
I guess this should not be in the commit? Looks like the traces of a merge conflict?

Reply:
Yes it does. thanks for seeing that. ill fix it soon